src/lib-csharp/Locators/WindowsVelopackLocator.cs

```diff
@@ -108,6 +115,15 @@ public WindowsVelopackLocator(string currentProcessPath, uint currentProcessId,
                     Channel = manifest.Channel;
                 }
             }
+
+            if (Log == null) {
+                try {
+                    Log = new FileVelopackLogger(Path.Combine(AppContext.BaseDirectory, "velopack.log"), currentProcessId);
```

Comment:
I am wondering if this will be a good base directory after we get all users install working. Perhaps this might be another place to leverage `Path.GetTempPath`?